00_Base/src/config/ocpi.server.config.ts

```diff
@@ -1,209 +0,0 @@
-import { IsBoolean, IsInt, IsNotEmpty, IsNumber, IsOptional, IsString, Min, ValidateNested, } from 'class-validator';
-import { Service } from 'typedi';
-import { Type } from 'class-transformer';
-import { Enum } from '../util/decorators/enum';
-import 'reflect-metadata';
-
-@Service()
-class SequelizeConfig {
-  @IsString()
-  @IsNotEmpty()
-  host!: string;
-
-  @IsInt()
-  @Min(0)
-  @IsNotEmpty()
-  port!: number;
-
-  @IsString()
-  @IsNotEmpty()
-  database!: string;
-
-  @IsOptional()
-  dialect?: any;
-
-  @IsString()
-  @IsNotEmpty()
-  username!: string;
-
-  @IsString()
-  @IsNotEmpty()
-  password!: string;
-
-  @IsString()
-  @IsNotEmpty()
-  storage!: string;
-
-  @IsBoolean()
-  @IsOptional()
-  sync?: boolean;
-
-  @IsBoolean()
-  @IsOptional()
-  alter?: boolean;
-
-  @IsNumber()
-  @IsOptional()
-  maxRetries?: number;
```

Comment:
Seems like this and `retryDelay` never made it to any other config.

Causing build errors on my end.

```
00_Base/src/util/sequelize.ts:71:51 - error TS2339: Property 'maxRetries' does not exist on type 'SequelizeConfig'.

71     const maxRetries = this.config.data.sequelize.maxRetries ?? 5;
                                                     ~~~~~~~~~~

00_Base/src/util/sequelize.ts:72:51 - error TS2339: Property 'retryDelay' does not exist on type 'SequelizeConfig'.

72     const retryDelay = this.config.data.sequelize.retryDelay ?? 5000;
```